toast-tk-runtime/src/main/java/io/toast/tk/runtime/block/TestBlockRunner.java

```diff
@@ -134,7 +175,7 @@ private ITestResult doLocalActionCall(final ActionAdaptaterLocator actionAdaptat
 				result = handler.result(output, expected);
 			}
 		} catch (final Exception e) {
-			result = handleInvocationError(e);
+			result = handleInvocationError(e, actionMethod, actionAdaptaterLocator);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "handleInvocationError()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)